### ISSUE TITLE
[1.3-beta.16] Update todos example - Conflict: Constraint ddp@1.2.2 is not satisfied by ddp 1.2.2-beta.16

### ISSUE THREAD

**Issue by D1no** [meteor/meteor#6447]
When updating the [todos example](https://github.com/meteor/todos/tree/master) as a try-out, the ddp package holds back the upgrade. See here:

```shell
meteor update --release 1.3-beta.16
Update to release METEOR@1.3-beta.16 is impossible:
While selecting package versions:             
error: Conflict: Constraint ddp@1.2.2 is not satisfied by ddp 1.2.2-beta.16.
Constraints on package "ddp":
* ddp@=1.2.2-beta.16 <- top level
* ddp@1.2.2-beta.16 <- meteor-base 1.0.1-beta.16
* ddp@1.2.2-beta.16 <- livedata 1.0.15-beta.16 <- meteor-base 1.0.1-beta.16
* ddp@1.2.2-beta.16 <- autoupdate 1.2.5-beta.16 <- avital:mocha 2.1.0_9
* ddp@1.2.2-beta.16 <- mongo 1.1.4-beta.16
* ddp@1.2.2-beta.16 <- allow-deny 1.0.1-beta.16 <- mongo 1.1.4-beta.16
* ddp@1.2.2-beta.16 <- accounts-password 1.1.5-beta.16
* ddp@1.2.2-beta.16 <- accounts-base 1.2.3-beta.16 <- accounts-password 1.1.5-beta.16
* ddp@1.1.0 <- simple:rest 1.1.0
* ddp@1.2.2-beta.16 <- force-ssl 1.0.8-beta.16 <- app-prod-security 0.0.0
* ddp@1.2.2 <- mdg:validated-method 1.0.1
* ddp@1.2.2 <- avital:mocha 2.1.0_9

This project is at the latest release which is compatible with your current package constraints.
```

**Comment by siegesmund** [comment]
I got this also, on an app previously running beta 12.

**Comment by jamiedumont** [comment]
I also got this on an app running beta 8.

**Comment by marbemac** [comment]
In case it helps - similar issue, different package, so it looks like it's not specific to ddp. Works on beta12 and earlier, and there is no mention of "ground:util 0.1.14" anywhere that I can find.

```
meteor update --release 1.3-beta.16
Update to release METEOR@1.3-beta.16 is impossible:
While selecting package versions:
error: Conflict: Constraint ground:util@0.1.17 is not satisfied by ground:util 0.1.14.
Constraints on package "ground:util":
* ground:util@0.1.17 <- ground:db 0.3.15 <- stoplight:core 0.0.1
```

**Comment by skirunman** [comment]
Getting similar update error.

```
meteor update --release 1.3-beta.16
Update to release METEOR@1.3-beta.16 is impossible:
While selecting package versions:             
error: Conflict: Constraint reactive-var@1.0.6 is not satisfied by reactive-var 1.0.6-beta.16.
Constraints on package "reactive-var":
* reactive-var@=1.0.6-beta.16 <- top level
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- accounts-base 1.2.3-beta.16 <- accounts-oauth
1.1.9-beta.16 <- accounts-salesforce 1.0.0
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- accounts-base 1.2.3-beta.16 <- accounts-password
1.1.5-beta.16
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- aldeed:autoform 5.8.1
* reactive-var@1.0.1 <- iron:dynamic-template 1.0.12 <- iron:controller 1.0.12 <- iron:router 1.0.12
* reactive-var@1.0.2 || 1.0.3 || 1.0.3 <- aldeed:autoform 5.8.1
* reactive-var@1.0.3 <- dispatch:reload-on-resume 0.0.2
* reactive-var@1.0.6 <- clozer:google-map 0.0.2
* reactive-var@1.0.3 <- dburles:google-maps 1.1.5 <- clozer:google-map 0.0.2
* reactive-var@1.0.6 <- clozertech:universe-autoform-select 0.1.15
* reactive-var@1.0.3 || 1.0.5 <- utilities:avatar 0.9.2
```

**Comment by jamielob** [comment]
Yep similar error for .16

```
While selecting package versions:
error: Conflict: Constraint reactive-var@1.0.6 is not satisfied by reactive-var 1.0.6-beta.16.
Constraints on package "reactive-var":
* reactive-var@=1.0.6-beta.16 <- top level
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- accounts-base 1.2.3-beta.16 <- accounts-oauth 1.1.9-beta.16 <- jamielob:accounts-facebook-cordova 0.0.2
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- accounts-base 1.2.3-beta.16 <- constellation:console 1.4.0
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- blaze-html-templates 1.0.1-beta.16
* reactive-var@1.0.3 <- constellation:console 1.4.0
* reactive-var@1.0.3 <- babrahams:editable-json 0.6.3 <- constellation:console 1.4.0
* reactive-var@1.0.3 <- kadira:flow-router 2.10.1
* reactive-var@1.0.6 <- jazzdragon:reactive-var-helper 1.0.1
* reactive-var@1.0.3 <- dburles:google-maps 1.1.5
* reactive-var@1.0.1 <- mdg:camera 1.1.5 <- okland:camera-ui 0.0.2
* reactive-var@1.0.3 <- lbee:moment-helpers 1.2.1
```

**Comment by seifip** [comment]
Getting a similar error:

```
Update to release METEOR@1.3-beta.16 is impossible:
While selecting package versions:
error: Conflict: Constraint blaze-html-templates@1.0.1 is not satisfied by blaze-html-templates
1.0.1-beta.16.
Constraints on package "blaze-html-templates":
* blaze-html-templates@=1.0.1-beta.16 <- top level
* blaze-html-templates@1.0.1 <- zodiase:mdl-prompt 0.0.1

Conflict: Constraint reactive-var@1.0.6 is not satisfied by reactive-var 1.0.6-beta.16.
Constraints on package "reactive-var":
* reactive-var@=1.0.6-beta.16 <- top level
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- accounts-base 1.2.3-beta.16 <-
accounts-oauth 1.1.9-beta.16 <- auth0:lock 0.5.0
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- accounts-base 1.2.3-beta.16 <- auth0:lock
0.5.0
* reactive-var@1.0.6-beta.16 <- blaze 2.1.4-beta.16 <- kadira:blaze-layout 2.3.0
* reactive-var@1.0.3 <- kadira:flow-router 2.10.1
* reactive-var@1.0.6 <- zodiase:mdl-prompt 0.0.1
* reactive-var@1.0.3 <- meteortoys:toykit 2.2.1 <- msavin:mongol 1.6.2

This project is at the latest release which is compatible with your current package constraints.
```

**Comment by tmeasday** [comment]
I've released `avital:mocha@2.1.0_10` which resolves this issue (if that's the only constraint you have), although you will probably run into this other issue with it: #6453

**Comment by tmeasday** [comment]
We should really just bump the ddp version to 1.2.3-beta.16 or so

**Comment by benjamn** [comment]
This is fixed in `1.3-rc.1`, since I bumped the version to `1.2.3-rc.1`.
